src/gettingstarted/cli.md

```diff
@@ -97,3 +97,28 @@ Once Bash for Windows is installed, you can install the Platform.sh CLI with the
 ```bash
 curl -sS https://platform.sh/cli/installer | php
 ```
+
+## API Tokens
+
+The CLI can also be used from CI services or other automation tools, and supports an API Token authentication option for that purpose.
+
+An API token can be created through the UI.  Go to the "User" page from your accounts drop-down, then select the "Account Settings" tab, then the "API Tokens".
+
+Click the "Create an API Token" link.  You may be asked to reverify your password, then enter a unique application name to identify the token.
+
+![API Token screen](/images/api-token-list.png)
+
+After creating the token, you may view it by clicking the "view" link next to the token name.  You will be asked to reverify your password as well.
```

Comment:
Also, directly after creating the token, you'll be shown the token value in a temporary message (a `drupal_set_message()`, if you see what I mean). To see it again, as you say, you will need to click "view" and you will need to re-enter your password.

Reply:
I completely missed that because it's all the way at the top of the page. :smile: Will update.